packages/xrpl/snippets/src/bridgeTransfer.ts

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable @typescript-eslint/consistent-type-assertions -- needed here */
+/* eslint-disable no-await-in-loop -- needed here */
+import {
+  AccountObjectsRequest,
+  LedgerEntry,
+  Client,
+  XChainAccountCreateCommit,
+  XChainBridge,
+  XChainCommit,
+  XChainCreateClaimID,
+  xrpToDrops,
+  Wallet,
+  TransactionMetadata,
+} from '../../src'
+import { CreatedNode } from '../../src/models/transactions/metadata'
+
+async function sleep(sec: number): Promise<void> {
+  return new Promise((resolve) => {
+    setTimeout(resolve, sec * 1000)
+  })
+}
+
+const lockingClient = new Client(
+  'wss://sidechain-net1.devnet.rippletest.net:51233',
+)
+const issuingClient = new Client(
+  'wss://sidechain-net2.devnet.rippletest.net:51233',
+)
+const MAX_LEDGERS_WAITED = 5
+const LEDGER_CLOSE_TIME = 4
+
+void bridgeTransfer()
+
+async function bridgeTransfer(): Promise<void> {
+  await lockingClient.connect()
+  await issuingClient.connect()
+  const lockingChainDoor = 'rMAXACCrp3Y8PpswXcg3bKggHX76V3F8M4'
+
+  const accountObjectsRequest: AccountObjectsRequest = {
+    command: 'account_objects',
+    account: lockingChainDoor,
+    type: 'bridge',
+  }
+  const bridgeData = (await lockingClient.request(accountObjectsRequest)).result
+    .account_objects[0] as LedgerEntry.Bridge
+  const bridge: XChainBridge = bridgeData.XChainBridge
+  console.log(bridge)
+
+  const { wallet: wallet1 } = await lockingClient.fundWallet()
+  const wallet2 = Wallet.generate()
+
+  console.log(
+    `Creating ${wallet2.classicAddress} on the issuing chain via the bridge...`,
+  )
+
+  const fundTx: XChainAccountCreateCommit = {
+    TransactionType: 'XChainAccountCreateCommit',
+    Account: wallet1.classicAddress,
+    XChainBridge: bridge,
+    SignatureReward: bridgeData.SignatureReward,
+    Destination: wallet2.classicAddress,
+    Amount: (
+      parseInt(bridgeData.MinAccountCreateAmount as string, 10) * 2
+    ).toString(),
+  }
+  const fundResponse = await lockingClient.submitAndWait(fundTx, {
+    wallet: wallet1,
+  })
+  console.log(fundResponse)
+
+  // wait for the attestation to go through
+  let ledgersWaited = 0
+  let initialBalance = '0'
+  while (ledgersWaited < MAX_LEDGERS_WAITED) {
+    await sleep(LEDGER_CLOSE_TIME)
+    try {
+      initialBalance = await issuingClient.getXrpBalance(wallet2.classicAddress)
+      console.log(
+        `Wallet ${wallet2.classicAddress} has been funded with a balance of ${initialBalance} XRP`,
+      )
+      break
+    } catch (_error) {
+      ledgersWaited += 1
+      // eslint-disable-next-line max-depth -- needed here
```

Comment:
Would it be worth disabling linting rules for snippets to avoid lines like this which are distracting from the point of the code? (Maybe it's not worth the trade off?)

Reply:
I can disable it at the top if you think that'll be easier to read? Linting rules are here to make sure the code is more readable, which is also the point of the code.